awx/api/serializers.py

```diff
@@ -4490,62 +4490,53 @@ class Meta:
             'instance_groups',
-            'execution_environment' 'scm_branch',
-            'verbosity' 'forks' 'char_prompts',
+            'execution_environment',
+            'scm_branch',
+            'verbosity',
+            'forks',
+            'char_prompts',
             'diff_mode',
             'extra_data',
             'job_slice_count',
             'job_tags',
-            'job_type' 'skip_tags',
+            'job_type',
+            'skip_tags',
             'survey_passwords',
             'timeout',
             # these are related objects and we need to add extra validation for them in the parent BulkJobLaunchSerializer
             #
         )
 
 
-class BulkJobLaunchSerializer(serializers.Serializer):
+class BulkJobLaunchSerializer(BaseSerializer):
     name = serializers.CharField(max_length=512, write_only=True, required=False)  # limited by max name of jobs
     jobs = BulkJobNodeSerializer(many=True, allow_empty=False, write_only=True, max_length=1000)
-    description: serializers.CharField(write_only=True, required=False, allow_blank=False)
-    extra_vars: serializers.CharField(write_only=True, required=False, allow_blank=False)
+    description= serializers.CharField(write_only=True, required=False, allow_blank=False)
+    extra_vars= serializers.CharField(write_only=True, required=False, allow_blank=False)
     organization = serializers.PrimaryKeyRelatedField(
         queryset=Organization.objects.all(),
         required=False,
         default=None,
         allow_null=True,
         help_text=_('Inherit permissions from organization roles.'),
     )
-    # inventory: "",  # Here we can use PrimaryKeyRelatedField so it will automagically do rbac/turn into object
-    limit: serializers.CharField(write_only=True, required=False, allow_blank=False)
-    scm_branch: serializers.CharField(write_only=True, required=False, allow_blank=False)
+    inventory= serializers.PrimaryKeyRelatedField(queryset=Inventory.objects.all(), required=False, write_only=True)
+    limit= serializers.CharField(write_only=True, required=False, allow_blank=False)
+    scm_branch= serializers.CharField(write_only=True, required=False, allow_blank=False)
+    # not implemented yet
     # webhook_service: null,  # Here we can use PrimaryKeyRelatedField so it will automagically do rbac/turn into object, I think, I'm actually not sure how to use this
     # webhook_credential: null,  # Here we can use PrimaryKeyRelatedField so it will automagically do rbac/turn into object  I think, I'm actually not sure how to use this
-    skip_tags: serializers.CharField(write_only=True, required=False, allow_blank=False)
-    job_tags: serializers.CharField(write_only=True, required=False, allow_blank=False)
-    is_bulk_job: serializers.BooleanField(default=True)
+    skip_tags= serializers.CharField(write_only=True, required=False, allow_blank=False)
+    job_tags= serializers.CharField(write_only=True, required=False, allow_blank=False)
```

Comment:
I think you probably need to have a venv with `black` installed and run `make black` to make this formatting compliant with the rules they hve on the PR checks

also, they use flake8 but I've never figured out the make command. you can check the errors on the ci checks or you can figure out which make command does that